File: libre-asi/src/lib/models/Interviewer.ts
interface Interviewer {
	ID?: number;
	CreatedAt?: Date;
	UpdatedAt?: Date;
	email?: string;
	username?: string;
	password?: string;
	firstName?: string;
	lastName?: string;
	firstSurname?: string;
	lastSurname?: string;
	birthdate?: Date;
	age?: number;
	personalID?: string;
}

export default Interviewer;

File: api/pkg/view/interviewer.go
package view

type Interviewer struct {
	ID                 uint   `json:"ID"`
	Email              string `json:"email"`
	Username           string `json:"username"`
	Password           string `json:"password"`
	NeedsPasswordReset bool   `json:"needsPasswordReset"`
	FirstName          string `json:"firstName"`
	LastName           string `json:"lastName"`
	FirstSurname       string `json:"firstSurname"`
	LastSurname        string `json:"lastSurname"`
	Birthdate          string `json:"birthdate"`
	Age                int    `json:"age"`
	PersonalID         string `json:"personalID"`
	RMA                string `json:"rma"`
}
